src/index.html

```diff
@@ -8,6 +8,108 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <section class="content">
+      <div class="stars stars--0">
+        <div class="stars__star">
+        </div>
+
+        <div class="stars__star">
+        </div>
+
+        <div class="stars__star">
+        </div>
+
+        <div class="stars__star">
+        </div>
+
+        <div class="stars__star">
+        </div>
+      </div>
```

Comment:
```suggestion
      <div class="stars stars--0">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
```
There is no need to add empty lines between empty children of one block